docs/tutorials/accelerate-pytorch/resnet-inferencing.md

```diff
@@ -0,0 +1,296 @@
+---
+nav_exclude: true
```

Comment:
Why exclude from the nav?

Reply:
Not sure, just followed other examples, but I can remove it